nservicebus/gateway/index.md

```diff
@@ -13,14 +13,14 @@ The purpose of the gateway is to provide durable fire-and-forget messaging with
 The gateway comes into play where the use of regular queued transports for communication are not possible i.e. when setting up a VPN connection is not an option. The reason for not using a VPN could be security concerns, bandwidth limitations, latency problems, high availability constraints, etc.
 
 
-## When not to use the gateway
+## When not avoid using the gateway
```

Comment:
> When not avoid using the gateway

That changes does seem correct @SimonCropp. Did you mean "When to avoid using the gateway"?